docs/oauth2-examples-azure/index.md

```diff
@@ -157,27 +157,21 @@ Update it with the following values:
 * **Application ID** associated to the app that you registered in Azure AD
 * Value of the **jwks_uri** key from `https://login.microsoftonline.com/{TENANT_ID}/v2.0/.well-known/openid-configuration`
 
-```bash
-$ vi rabbitmq.config
-[
-  {rabbit, [
-   {auth_backends, [rabbit_auth_backend_oauth2, rabbit_auth_backend_internal]}
-  ]},
-  {rabbitmq_management, [
-     {oauth_enabled, true},
-     {oauth_client_id, "PUT YOUR AZURE AD APPLICATION ID"},
-     {oauth_provider_url, "https://login.microsoftonline.com/AZURE_AD_TENANT_ID"}
- ]},
- {rabbitmq_auth_backend_oauth2, [
-   {resource_server_id, <<"PUT YOUR AZURE AD APPLICATION ID">>},
-   {extra_scopes_source, <<"roles">>},
-   {key_config, [
-     {jwks_url, <<"PUT YOUR AZURE AD JWKS URI VALUE">>}
-   ]}
- ]}
-].
+```ini
+auth_backends.1 = rabbit_auth_backend_oauth2
+auth_backends.2 = rabbit_auth_backend_internal
+
+management.oauth_enabled = true
+management.oauth_client_id = <YOUR_AZURE_AD_TENANT_ID>
+management.oauth_scopes = openid profile rabbitmq.tag:administrator
+management.oauth_provider_url = https://login.microsoftonline.com/<YOUR_AZURE_AD_TENANT_ID>
+
+auth_oauth2.resource_server_id = <YOUR_AZURE_AD_TENANT_ID>
+auth_oauth2.additional_scopes_key = roles
+auth_oauth2.jwks_url = <<"PUT YOUR AZURE AD JWKS URI VALUE">>
```

Comment:
Using a binary here may confuse some people. `rabbitmq.conf` does not accept the binary syntax, even though a mapping type can be used to produce a binary out of a value.

Reply:
It was unfortunate that i chose << " " >> :) I will use <PUT YOUR AZURE AD JWKS URI VALUE> instead. I did not mean erlang binary value.